Sources/ForageSDK/Foundation/Network/LDManager.swift

```diff
@@ -11,12 +11,79 @@ import LaunchDarkly
 
 internal protocol LDClientProtocol {
     func doubleVariationWrapper(forKey key: String, defaultValue: Double) -> Double
+    func jsonVariationWrapper(forKey key: String, defaultValue: LDValue) -> LDValue
 }
 
 extension LDClient: LDClientProtocol {
     func doubleVariationWrapper(forKey key: String, defaultValue: Double) -> Double {
         return self.doubleVariation(forKey: key, defaultValue: defaultValue)
     }
+    
+    func jsonVariationWrapper(forKey key: String, defaultValue: LDValue) -> LDValue {
+        return self.jsonVariation(forKey: key, defaultValue: defaultValue)
+    }
+}
+
+extension LDValue {
+    
+    /// Converts the original LDValue returned from the polling feature flag to an Array of Ints.
+    internal func convertToIntArray() -> [Int] {
+        let intervalArrayAsLDValue = self.extractArray(pollingFlagObj: self)
+        let intervalArrayAsArray = self.convertToArray(pollingArray: intervalArrayAsLDValue)
+        return self.convertArrayValsToInts(arrayOfInts: intervalArrayAsArray)
+    }
+    
+    /// Inspects the original LDValue returned from the polling feature flag and extracts the value mapped to INTERVALS
+    ///
+    /// - Parameters:
+    ///   - pollingFlagObj: The `LDValue` representing a JSON object.
+    internal func extractArray(pollingFlagObj: LDValue) -> LDValue {
+        let defaultLDValueArray: LDValue = LDValue(arrayLiteral: [])
+        switch pollingFlagObj {
+        case .object(let dictValue):
+            for (key, value) in dictValue {
+                if (key == INTERVALS) {
```

Comment:
Note for this ticket or future cleanup:

```swift
key == INTERVALS
```
is specific to the shape of the polling-intervals flag. We should parameterize the `key` and keep the `LDValue` extension more generic so that this logic can be re-used for future flag values.

```swift
    internal func extractArray(targetKey: String, flagObject: LDValue) -> LDValue {
```